### DISCUSSION TITLE
Windows 11 Insider Preview workaround

### DISCUSSION THREAD

**Discussion by Armin2208** [AutoDarkMode/Windows-Auto-Night-Mode#435] [Q&A] [answered]
Update: the pushed a fix into the Auto Dark Mode Beta channel, which you can enter in settings.

With the current Windows 11 22H2 Insider previews there is a problem with the theme switch. For example, the taskbar and file explorer don't adapt. Currently we didn't investigate in a fix because an Insider preview isn't a final product so things could still change or revert.

But there is luckily a workaround! Use the Windows theme file switch, it still works as expected.

![image](https://user-images.githubusercontent.com/32982430/158464412-9420a54f-a135-40ff-b008-5bde57b81e00.png)

**Comment by jmoo2** [comment]
Thanks for your tip. Works great!

**Comment by lambdakilo** [comment, marked as answer]
You seem to have fixed this already. Maybe mark this answered?
